src/components/description-data/styles.scss

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
Ik ben pro om al deze comments weg te doen, maar ik weet dat dit nog bijna nergens is gedaan. Up to you!